src/templates/app/config/database.yml.ecr

```diff
@@ -0,0 +1,6 @@
+<%= @database %>:
+<% if @database == "mysql" -%>
```

Comment:
Technically I think that it should have defaults for both pg and mysql and eventually sqlite3 in the database.yml. Environment VAR DATABASE_URL should be loaded if it exists regardless of if something else is set in the database.yml.